tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -189,9 +189,11 @@ spec:
           value: "false"
       computeResources:
         requests:
-          memory: 300Mi
+          cpu: 250m
+          memory: 500Mi
         limits:
-          memory: 300Mi
+          cpu: 2
```

Comment:
I hear CPU limits is discouraged? See changes in build-definitions: https://github.com/redhat-appstudio/build-definitions/pull/673

Reply:
Actually... it's not clear to me why we are setting CPU requests/limits on the PR at all. The reported issue seems to be about memory.

Reply:
The reported issue _is_ about memory, however we are seeing cluster-wide issues due to tasks in general not setting cpu _requests_: over-subscribed nodes.

I'll remove the cpu limits.